File: levelledmobs-plugin-bukkit/src/main/java/me/lokka30/levelledmobs/bukkit/integration/translation/TranslationProvider.java
package me.lokka30.levelledmobs.bukkit.integration.translation;

import org.bukkit.entity.EntityType;
import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

public interface TranslationProvider {

    @Nullable
    String getTranslatedEntityName(
        final @NotNull EntityType entityType,
        final @NotNull String locale
    );

}

File: levelledmobs-plugin-bukkit/src/main/kotlin/me/lokka30/levelledmobs/bukkit/integrations/type/TranslationProvider.kt
package me.lokka30.levelledmobs.bukkit.integrations.type

import org.bukkit.entity.EntityType

/*
FIXME Comment
 */
interface TranslationProvider {

    /*
    FIXME Comment
     */
    fun getTranslatedEntityName(entityType: EntityType, locale: String): String

}